content/en/error_tracking/frontend/mobile/ios.md

```diff
@@ -1,8 +1,746 @@
+
+To be compliant with the GDPR regulation, the iOS SDK requires the tracking consent value at initialization.
+
+The `trackingConsent` setting can be one of the following values:
+
+1. `.pending`: The iOS SDK starts collecting and batching the data but does not send it to Datadog. The iOS SDK waits for the new tracking consent value to determine what to do with the batched data.
+2. `.granted`: The iOS SDK starts collecting the data and sends it to Datadog.
+3. `.notGranted`: The iOS SDK does not collect any data, and doesn't send logs, traces, or events to Datadog.
+
+To **change the tracking consent value** after the iOS SDK is initialized, use the `Datadog.set(trackingConsent:)` API call. The iOS SDK changes its behavior according to the new value.
+
+For example, if the current tracking consent is `.pending`:
+
+- If you change the value to `.granted`, the iOS SDK sends all current and future data to Datadog.
+- If you change the value to `.notGranted`, the iOS SDK wipes all current data and does not collect future data.
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Sample session rates" level="h4" expanded=false id="sample-session-rates" %}}
+
+To control the data your application sends to Datadog, you can specify a sampling rate for sessions while [initializing the iOS SDK][14]. The rate is a percentage between 0 and 100. By default, `sessionSamplingRate` is set to 100 (keep all sessions).
+
+For example, to only keep 50% of sessions, use:
+
+{{< tabs >}}
+{{% tab "Swift" %}}
+```swift
+let configuration = RUM.Configuration(
+    applicationID: "<application id>",
+    sessionSampleRate: 50
+)
+```
+{{% /tab %}}
+{{% tab "Objective-C" %}}
+```objective-c
+DDRUMConfiguration *configuration = [[DDRUMConfiguration alloc] initWithApplicationID:@"<application id>"];
+configuration.sessionSampleRate = 50;
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Send data when device is offline" level="h4" expanded=false id="sending-data-device-offline" %}}
+
+The iOS SDK ensures data remains available when an end user's device is offline. In cases of low-network areas, or when the device battery is too low, the device first stores all events in batches. It sends them as soon as the network is available and the battery is high enough to ensure the iOS SDK does not impact the end user's experience. If the network is not available while your application is in the foreground, or if a data upload fails, the device keeps the batch until it can complete a successful upload.
+
+This means that even if users open your application while offline, no data is lost.
+
+**Note**: To ensure the iOS SDK does not use too much disk space, the device automatically discards the data on the disk if the data gets too old.
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Add watchdog termination reporting" level="h4" expanded=false id="add-watchdog-termination-reporting" %}}
```

Comment:
I just noticed that we use both `termination reporting` and `terminations reporting` in this section. Might be worth choosing one and sticking with it? Unless we use the plural in the UI somewhere, I think `termination reporting` sounds a smidge less awkward to me, but it's totally up to you.

Reply:
thanks for noticing this! after a google search, i determined "watchdog terminations reporting" would be the most appropriate here.